src/drivers/intel/ssp/ssp.c

```diff
@@ -975,6 +975,11 @@ static void ssp_early_start(struct dai *dai, int direction)
 
 	key = k_spin_lock(&dai->lock);
 
+	if (direction == DAI_DIR_CAPTURE) {
+		/* RX fifo must be cleared before start */
+		ssp_empty_rx_fifo(dai);
+	}
```

Comment:
niptick:
```c
/* Before starting a capture the RX FIFO must be cleared */
if (direction == DAI_DIR_CAPTURE)
	ssp_empty_rx_fifo(dai);
```

Reply:
Changed (although noting that in Zephyr, braces are mandatory here, trying to learn this style as this gets us rejects on Zephyr side all the time).

Reply:
Well, if you have the comment inside of the if, then yes, the braces help, but I suggested to move it _outside_ of the if:
```c
/* Before starting a capture the RX FIFO must be cleared */
if (direction == DAI_DIR_CAPTURE)
	ssp_empty_rx_fifo(dai);
```
Does Zephyr mandates that braces are mandatory in all instance?